examples/experimental/openvino/mobilenet_v2/main.py

```diff
@@ -63,21 +61,28 @@ def transform_fn(data_item):
 
     # Wrap framework-specific data source into the `nncf.Dataset` object.
     calibration_dataset = nncf.Dataset(data_source, transform_fn)
+
+    # Quantization of the OpenVINO model. The `quantize` method expects an
+    # OpenVINO model as input and returns an OpenVINO model that has been
+    # quantized using the calibration dataset.
     quantized_model = nncf.quantize(ov_model, calibration_dataset)
 
     # Step 5: Save the quantized model.
     ir_qmodel_xml = MODEL_DIR / 'mobilenet_v2_quantized.xml'
```

Comment:
this is the way how to make this code brief:
```
dst_path = 
openvino.runtime.serialize(quantized_model, dst_path, dst_path.replace(".xml", ".bin"))
```

Reply:
Did you mean these changes?
```
    dst_path = str(MODEL_DIR / 'mobilenet_v2_quantized.xml')
    ov.serialize(quantized_model, dst_path, dst_path.replace('.xml', '.bin'))
    print('The quantized model has been saved in:')
    print(f'XML file: {dst_path}')
    print(f'BIN file: {dst_path.replace(".xml", ".bin")}')
```